### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**
Finished my homework.

### CODE REVIEW

**Review by naresh699 — APPROVED** [review]

File: javascript/javascript2/week3/index1.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Warmup</title>
+    <script src="main1.js"></script>
+  </head>
+  <body>
+    <button onclick="notThisFunctionName(5, 'Called after 5 seconds')">
```
[the remark this answers is not in the document]
Comment (Atefeh-Behroozi) [reply javascript/javascript2/week3/index1.html:11]: Ok Naresh, Sure.

File: javascript/javascript2/week3/main1.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  timeout = setTimeout(() => {
+    clicks = 0;
+  }, 500);
+}
+
+document.addEventListener("click", checkDoubleClick);
+
+//9
+function jokeCreator(shouldTellFunnyJoke, logFunnyJoke, logBadJoke) {
+if (shouldTellFunnyJoke) {
```
[the remark this answers is not in the document]
Comment (Atefeh-Behroozi) [reply javascript/javascript2/week3/main1.js:68]: Ok, I'II do it.

[the remark this answers is not in the document]
Comment (Atefeh-Behroozi) [reply javascript/javascript2/week3/main1.js:68]: Thank you so much for your feedback.

**Comment by Atefeh-Behroozi** [comment]
Thank you for feedback. I added 'class or id' in button and 'addEventListener' and try JS ternary condition.

**Pull request merged**